intelmq/bots/parsers/shadowserver/_config.py

```diff
@@ -2881,8 +2887,8 @@ def scan_exchange_identifier(field):
     ('SSL-POODLE-Vulnerable-Servers', 'scan_ssl_poodle', ssl_poodle_vulnerable_servers),
     ('Sandbox-URL', 'cwsandbox_url', sandbox_url),
     ('Sinkhole-DNS', 'sinkhole_dns', sinkhole_dns),
-    ('Sinkhole-Events', 'event4_sinkhole', event46_sinkhole),
-    ('Sinkhole-Events', 'event6_sinkhole', event46_sinkhole),
+    ('Sinkhole-Events IPv4', 'event4_sinkhole', event46_sinkhole),
+    ('Sinkhole-Events IPv6', 'event6_sinkhole', event46_sinkhole),
```

Comment:
```suggestion
    ('Sinkhole-Events', 'event4_sinkhole', event46_sinkhole),
    ('Sinkhole-Events IPv4', 'event4_sinkhole', event46_sinkhole),
    ('Sinkhole-Events IPv6', 'event6_sinkhole', event46_sinkhole),
```
We can also suppoort the unversioned, but simpler and documented, feed name